File: Python/9thOct.py
# Python code to find GCD of factorials of 
# two numbers. 
import math 
  
def gcdOfFactorial(m, n) : 
    return math.factorial(min(m, n)) 
  
# Driver code 
n = int(input("Enter an number >>>"))
m = int(input("Enter an number >>>"))

print(gcdOfFactorial(m, n))

File: JavaScript/9thOct.js
function factorial(n){
    
    if(n == 0 || n == 1){
         //base case
          return 1;
    }else{
        //recursive case
        return n * factorial(n-1);
    }
}

var gcdOfFactorial = function(a, b) {
   if(a<b){
   		return a;
   }else{
   		return b;
   }
};

//output
console.log(gcdOfFactorial(factorial(5),factorial(120)));
